Package.swift

```diff
@@ -3,6 +3,7 @@ import PackageDescription
 
 let package = Package(
 	name: "Preferences",
+    defaultLocalization: "fr",
```

Comment:
This is incorrect.

Reply:
What is wrong with my code? the indentation or the sentence?

Reply:
Both. The default locale is not French.